server/graphql/general/src/queries/plugin.rs

```diff
@@ -0,0 +1,25 @@
+use async_graphql::*;
+use graphql_core::ContextExt;
+use service::plugin_files::PluginFileService;
+
+#[derive(PartialEq, Debug, SimpleObject)]
+pub struct PluginNode {
+    pub config: String,
+    pub name: String,
+    pub path: String,
+}
+
+pub fn get_plugins(ctx: &Context<'_>) -> Result<Vec<PluginNode>, Error> {
+    let settings = ctx.get_settings();
+    let plugins = PluginFileService::find_files(&settings.server.base_dir)?;
```

Comment:
Could you please convert this to our standard internal error?

```suggestion
    let plugins = PluginFileService::find_files(&settings.server.base_dir)
        .map_err(|err| StandardGraphqlError::InternalError(format!("{:?}", err)))?;
```

Reply:
thanks! I missed this before merging, so have added in a separate commit: 80609f917